gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/GateConfig.groovy

```diff
@@ -404,6 +407,17 @@ class GateConfig extends RedisHttpSessionConfiguration {
     return frb
   }
 
+  @Bean
```

Comment:
Do we want this enabled by default ..?

Reply:
👍 

Flipped this over to be enabled by default. It doesn't impact Netflix but I _suspect_ not everyone has the request logging enabled.